superset/databases/commands/update.py

```diff
@@ -94,11 +95,35 @@ def run(self) -> Model:
                 security_manager.add_permission_view_menu(
                     "schema_access", security_manager.get_schema_perm(database, schema)
                 )
+
+            if ssh_tunnel_properties := self._properties.get("ssh_tunnel"):
+                existing_ssh_tunnel = DatabaseDAO.get_ssh_tunnel(database.id)
+                if existing_ssh_tunnel is None:
+                    # We couldn't found an existing tunnel so we need to create one
+                    SSHTunnelDAO.create(
+                        {
+                            **ssh_tunnel_properties,
+                            "database_id": database.id,
+                        },
+                        commit=False,
+                    )
+                else:
+                    # We found an existing tunnel so we need to update it
+                    ssh_tunnel_model = SSHTunnelDAO.find_by_id(existing_ssh_tunnel.id)
+                    SSHTunnelDAO.update(
+                        ssh_tunnel_model,
+                        ssh_tunnel_properties,
+                        commit=False,
+                    )
+
             db.session.commit()
 
         except DAOUpdateFailedError as ex:
             logger.exception(ex.exception)
             raise DatabaseUpdateFailedError() from ex
+        except DAOCreateFailedError as ex:
+            logger.exception(ex.exception)
```

Comment:
the logger should log at the root api level

Reply:
nit, but if the except block on both of these errors, is the same, you might as well just catch both errors in the same block, no?